main.lua

```diff
@@ -0,0 +1,104 @@
+imgxKaizer=200
+imgyKaizer=0
+
+imgxC1 = 0
+imgyC1 = 100
+
+imgxC2 = 0
+imgyC2 = 200
+
+imgxC3 = 0
+imgyC3 = 300
+
+function love.load()
+    imageKaizer = love.graphics.newImage("Kaizer.JPG")
+    widthKaizer = imageKaizer:getWidth()
+    heightKaizer = imageKaizer:getHeight()    
+    
+    imageCar1 = love.graphics.newImage("Car1.JPG")
+    widthCar1 = imageCar1:getWidth()
+    heightCar1 = imageCar1:getHeight()
+
+    imageCar2 = love.graphics.newImage("Car2.JPG")
+    widthCar2 = imageCar2:getWidth()
+    heightCar2 = imageCar2:getHeight()
+
+    imageCar3 = love.graphics.newImage("Car3.JPG")
+    widthCar3 = imageCar3:getWidth()
+    heightCar3 = imageCar3:getHeight()
+    
+
+end
+
+function love.update(dt)
+    
+    imgxC1 = imgxC1 + 300 * dt
+
+    if imgxC1 > 1000 then
+        imgxC1 = 0
+    end
+
+    imgxC2 = imgxC2 - 300 * dt
+
+    if imgxC2 < 0 then
+        imgxC2 = 1000
+    end
+
+    imgxC3 = imgxC3 + 300 * dt
+
+    if imgxC3 > 1000 then
+        imgxC3 = 0
+    end
+
+	if love.keyboard.isDown("up") then
+        imgyKaizer = imgyKaizer - 500 * dt -- this would increment num by 100 per second
+    end
+
+	if love.keyboard.isDown("down") then
+        imgyKaizer = imgyKaizer + 500 * dt -- this would increment num by 100 per second
+    end
+
+	if love.keyboard.isDown("left") then
+        imgxKaizer = imgxKaizer - 500 * dt -- this would increment num by 100 per second
+    end
+
+	if love.keyboard.isDown("right") then
+		imgxKaizer = imgxKaizer + 500 * dt -- this would increment num by 100 per second
+    end
+    
+
+end
+ 
+function love.draw()
+	love.graphics.draw(imageKaizer, imgxKaizer, imgyKaizer)
+    love.graphics.draw(imageCar1, imgxC1, imgyC1)
+    love.graphics.draw(imageCar2, imgxC2, imgyC2)
+    love.graphics.draw(imageCar3, imgxC3, imgyC3)
+	love.graphics.print("Click and drag the cake around or use the arrow keys", 10, 10)
+	love.graphics.print(tonumber(imgxKaizer), 10, 20)
+    love.graphics.print(tonumber(imgyKaizer), 10, 30)
+    love.graphics.print(tonumber(widthKaizer), 10, 40)
+
+    if collision(imgxC1, imgyC1, widthCar1, heightCar1, imgxKaizer, imgyKaizer, widthKaizer, heightKaizer) then

```

Comment:
As a next step, you should group up your collision checks so that you can loop through all the items. For that you need to group your Car "objects" into a Cars collection.

Reply:
Placed sprites in assets folder and linked accordingly in love.load.